app/scripts/modules/core/src/header/SpinnakerHeader.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import { BindAll } from 'lodash-decorators';
 import { UISref, UISrefActive } from '@uirouter/react';
 import { UIRouterContext } from '@uirouter/react-hybrid';
+import { GlobalSearch } from '../search/global/GlobalSearch';
```

Comment:
We generally put imports into four sections:

1) import from react/angular
1b) import from 3rd party lib 

2) import from `@core/somemodule`

3) relatively import something from `../foo` or `./foo`

4) import from `./somefile.css`